src/components/calculator.js

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import '../calculator.css';
+
+export default function Calculator() {
```

Comment:
- [ ] Right now, your Calculator is a function-based component but according to the project requirements, it should be a class-based component. Kindly convert your component accordingly.